roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -74,6 +74,14 @@
       "tune_ballast_file": true
     },
     "pandaproxy": {},
-    "schema_registry": {}
+    "schema_registry": {
+        "schema_registry_api": [
+        {
+          "address": "0.0.0.0",
```

Comment:
might be worth making an issue about this not being settable to the value of advertised_ip

Reply:
It's basically the same reason you can't set the Kafka API listener to its advertised address - it might not be on the same machine. Basically:
* Copy whatever is in the `kafka_api` section, to schema_registry_api, but with the SR port (8081)
* If the Kafka API has an `authentication_method`, set `authentication_method: http_basic` on the SR listener
* Copy the contents of `kafka_api_tls` to `schema_registry_api_tls`

Reply:
Is there an equivalent of advertised_kafka_api for schema registry? advertised_schema_registry? How is a client expected to reach a private address when no LB is configured (as in our default most basic config)?

Reply:
> Is there an equivalent of advertised_kafka_api for schema registry?

No.

> How is a client expected to reach a private address when no LB is configured (as in our default most basic config)?

In exactly the same way you configure routing for the Kafka API (but with a different port).

You can get clever with a an HTTP Proxy, Load Balancer, TLS termination, whatever you like, but fundamentally the routing is the same. Expose a node port?

Reply:
It might be challenging to expose a NodePort on an ec2 instance 😅 

Since this is self hosted and our default demo is open everything with no intermediate elements we kind of need to be able to expose stuff that users have an expectation will be externally accessible for testing

Given that, 0.0.0.0 seems like the best fit for our requirements until an advertised_ shows up

Reply:
> Given that, 0.0.0.0 seems like the best fit for our requirements until an advertised_ shows up

an advertised is not needed, and will not turn up. Schema Registry is just an HTTP Server. Every instance is equivalent.

Having an advertised address for the Kafka API doesn't magically set up any routing.

Just set up the same listeners and routing as for the Kafka API. A single listener on `0.0.0.0` is probably fine, if you only need a single type of AuthN.